src/frontend/src/pages/home/home.tsx

```diff
@@ -1,7 +1,112 @@
+                            </Link>
+                        </Grid>
+                    </Grid>
+                </Toolbar>
+            </AppBar>
+
+
+            {/* new challenge button    */}
+
+
+            <Box p={3}>
+                <Grid
+                    container
+                    direction="column"
+                    justify="center"
+                    alignItems="center"
+                    spacing={3}
+                >
+
+                    <Grid item>
+                        <Link to="/new-challenge" style={{ textDecoration: 'none' }}>
+                            <Button variant="contained" color="primary" size="large">
+                                New Challenge
+                            </Button>
+                        </Link>
+                    </Grid>
+
+
+                    {/*     challenge list      */}
+                    <Grid item>
+                        <Typography variant="h6">
+                            Challenges from friends
+                        </Typography>
+                    </Grid>
+                    <Grid item>
+                        <Paper elevation={3}>
+                            <List component="nav" >
+                                <Link to="/view-challenge" style={{ textDecoration: 'none' }}>
+                                    <ListItem button divider>
+                                        <ListItemText primary="@William" />
+                                    </ListItem>
+                                </Link>
+                                <Link to="/view-challenge" style={{ textDecoration: 'none' }}>
+                                    <ListItem button divider>
+                                        <ListItemText primary="@Ryan" />
+                                    </ListItem>
+                                </Link>
+                                <Link to="/view-challenge" style={{ textDecoration: 'none' }}>
+                                    <ListItem button divider>
+                                        <ListItemText primary="@Amir" />
+                                    </ListItem>
+                                </Link>
+                                <Link to="/view-challenge" style={{ textDecoration: 'none' }}>
+                                    <ListItem button divider>
+                                        <ListItemText primary="@Lauren" />
+                                    </ListItem>
+                                </Link>
+                                <Link to="/view-challenge" style={{ textDecoration: 'none' }}>
+                                    <ListItem button>
+                                        <ListItemText primary="@Tina" />
+                                    </ListItem>
+                                </Link>
+                            </List>
+                        </Paper>
+                    </Grid>
+                </Grid>
+            </Box>
+        </Box>
+            )};
```

Comment:
```suggestion
    );
};

```
Some minor formatting changes